contrib/sssd.spec.in

```diff
@@ -199,6 +199,8 @@ Requires: sssd-common = %{version}-%{release}
 Requires: python3-sss = %{version}-%{release}
 Requires: python3-sssdconfig = %{version}-%{release}
 Requires: libsss_certmap = %{version}-%{release}
+# required by sss_analyze
+Requires: python3-systemd
```

Comment:
If `python3-systemd` is needed only for `journals` access?

Reply:
Yes that's correct, i'm not sure journald is working with tevent chain id however: see https://github.com/SSSD/sssd/pull/5666#issuecomment-879151580